kicad_mod/kicad_mod.py

```diff
@@ -179,24 +207,34 @@ def _savePad(self, data):
         output += ' '
         output += self._saveSize(data['size'], 'size')
         output += ' (drill {drill}) '.format(drill=data['drill'])
-        output += '(layers ' + ' '.join(data['layers']) + '))\r\n'
+        output += '(layers ' + ' '.join(data['layers']) + '))\n'
         return output
-
-
+        
+    def _saveModel(self):
+        
+        #model path
+        output = "(model {model}\n".format(model=self.model)
+        #model position
+        output += "\t(at (xyz {x} {y} {z}))\n".format(x=self.model_pos['x'],y=self.model_pos['y'],z=self.model_pos['z'])
+        output += "\t(scale (xyz {x} {y} {z}))\n".format(x=self.model_scale['x'],y=self.model_scale['y'],z=self.model_scale['z'])
+        output += "\t(rotate (xyz {x} {y} {z})))\n".format(x=self.model_rot['x'],y=self.model_rot['y'],z=self.model_rot['z'])
```

Comment:
replace ```\t``` with the correct amount of spaces (look onto files which were created by kicad which means 2 spaces per indention level, and the last closing bracket is located in a new line).
The script output should be the same as kicad does it.